src/py21cmfast/outputs.py

```diff
@@ -104,9 +109,21 @@ def _init_arrays(self):
         self.lowres_vz_2LPT = np.zeros(
             self.user_params.HII_tot_num_pixels, dtype=np.float32
         )
+
```

Comment:
Should we be initializing these _only_ if doing the `PERTURB_ON_HIGH_RES`?

Reply:
There are quite a few different arrays (in most classes) that should only be initialised under certain parameter combination. Issues #27, #63 and #69 discuss this to various degrees. At this point I am just aiming to get in all pieces of functionality, before going through and initialising only specific pieces.

Reply:
Yup, fair enough. We can address this when addressing those issues.